scripts/check_website.ts

```diff
@@ -29,8 +30,10 @@ try {
 
 const app = new App({ appId: Number(env.APP_ID), privateKey: env.PRIVATE_KEY });
 const octokit = await app.getInstallationOctokit(Number(env.INSTALLATION_ID));
+setOctokit(octokit);
 
 const me = await app.octokit.request("GET /app");
+if (!me.data) throw new Error(`Could not GET /app, returned ${me.data}`);
```

Comment:
What's the point of `returned ${me.data}` within `if (!me.data)`?

Reply:
It prints if it's null, undefined, or an empty string. But perhaps we could do
```suggestion
if (!me.data) throw new Error(`Could not GET /app, returned ${Deno.inspect(me)}`);
```
instead?

Reply:
Isn't there a field like `me.error`?

Reply:
There are both `me.status` and `me.error` properties according to https://github.com/octokit/request-error.js?tab=readme-ov-file#usage

Reply:
The suggestion above LGTM.